jbpm-designer-client/src/test/java/org/jbpm/designer/client/parameters/DesignerEditorParametersPublisherTest.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright 2015 JBoss Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+package org.jbpm.designer.client.parameters;
+
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Spy;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class DesignerEditorParametersPublisherTest {
+
+    @Mock
+    private DesignerEditorParametersPublisherView view;
+
+    @Spy
+    @InjectMocks
+    private DesignerEditorParametersPublisher publisher = new DesignerEditorParametersPublisher();
+
+    private Map<String, String> parameters;
+
+    @Before
+    public void clearParameters() {
+        parameters = new HashMap<String, String>();
+    }
+
+    @Test
+    public void testProcessSources() {
+        parameters.put(DesignerEditorParametersPublisher.PROCESS_SOURCE_KEY, "value");
+        publisher.publishProcessSources(parameters);
+
+        verify(view, times(1)).publishProcessSourcesInfo("value");
```

Comment:
@jhrcek cosmetics :) I think the way it is now seems fine. From readability of the test where the intent is to make sure the methods gets called and no more than 1 times this reads better.